programs/pkey/dh_genprime.c

```diff
@@ -71,7 +71,7 @@ int main(int argc, char **argv)
     mbedtls_ctr_drbg_init(&ctr_drbg);
     mbedtls_entropy_init(&entropy);
 
-    if (argc == 0) {
+    if (argc < 2) {
```

Comment:
No, that's not right. This program doesn't require an argument. Several others don't either. https://github.com/Mbed-TLS/mbedtls/issues/7100